File: 프로그래머스/lv1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(vector<string> seoul) {
    int f = -1;

    for(int i = 0; i < seoul.size(); i++)
    {
        if(seoul[i].compare("Kim") == 0) { f = i; break; }
    }
    
    string x = "";
    x += "김서방은 "; x += to_string(f); x += "에 있다";
    return x;
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        int posKim = 0;
        
        for(int i = 0; i < seoul.length; i++) {
            if(seoul[i].compareTo("Kim") == 0) {
                posKim = i;
                break;
            }
        }
        
        String answer = Integer.toString(posKim);
        return "김서방은 " + answer + "에 있다";
    }
}